### PR TITLE
class entities with backwards compatibility

### PR DESCRIPTION
**Pull request by valyrie97**

### CODE REVIEW

File: src/Nexus.js [hunk 1]
```diff
@@ -203,6 +203,14 @@ module.exports = function xGraph() {
 
 			}
 
+			function indirectEvalImp(entString) {
+				let imp = (1, eval)(entString);
+				console.dir(imp);
```
Comment (tjgionet) [inline src/Nexus.js]: Not sure we want be printing all of this out

**Review by tjgionet — COMMENTED** [review]
Not sure we want to be printing all of this.

File: src/Nexus.js (hunk 1 above)
Comment (valyrie97) [reply src/Nexus.js]: tru

**Review by tjgionet — APPROVED** [review]

**Comment by valyrie97** [comment]
@tjgionet Travis is passing. Nexe 2.0.0 rc 24 seems to be the most stable for now.

**Pull request merged**